### ISSUE TITLE
Browse Inflection tables opens Latin not Greek tables

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alpheios-core#344]
1) I changed page language to Greek in Options, I also changed Lookup to be Greek. I lookup a greek word ἅπασαν. if I click on Grammar it does open the Greek grammar, if I click on Inflection tables icon, it does still open the Latin Inflection icon. The text is in Latin.
2) same as above with a Greek text. The Browse Inflection Tables opens on Latin Inflection tables. 
Shouldn't it open the Greek inflection tables in expanded view?
see screenshot
![greek-inflection-tables](https://user-images.githubusercontent.com/41396793/82218688-45db7180-98ea-11ea-8441-9fb27320f3ed.JPG)

**Comment by balmas** [comment]
yes, probably. Before I decide priority, @monzug can you tell me if this is an issue for the current release?

**Comment by monzug** [comment]
also, when I switch from Syriac to Greek, the Browse inflection table is not expanded.
will try later. it was not working before.

**Comment by monzug** [comment]
yes, it's same in Release. so I guess it's looooow priority

**Comment by balmas** [comment]
ok, so yes let's hold off on this one.
